### PR TITLE
GG-23862 Sorted configuration properties. Fixed showing of not configured object.

### PR DESCRIPTION
**Pull request by vsisko**

### CODE REVIEW

File: modules/web-console/e2e/testcafe/page-models/pageSignin.js [hunk 1]
```diff
@@ -24,6 +25,7 @@ export const pageSignin = {
     selector: Selector('page-signin'),
     async login(email, password) {
         return await t
+            .expect(activeLoadingOverlay.exists).notOk()
```
Comment (Klaster1) [inline modules/web-console/e2e/testcafe/page-models/pageSignin.js]: How is this related to the task?

Comment (vsisko) [reply modules/web-console/e2e/testcafe/page-models/pageSignin.js]: In time of test development I notice that opening of Sign In page is longer than waiting time for input field.
